jobs/uaa-customized/templates/web/login.html.erb

```diff
@@ -0,0 +1,127 @@
+
+      <div th:if="${error}" class="govuk-error-summary" aria-labelledby="error-summary-title" role="alert" tabindex="-1" data-module="error-summary">
+        <h2 class="govuk-error-summary__title" id="error-summary-title">
+          There is a problem
+        </h2>
+        <div class="govuk-error-summary__body">
+          <ul class="govuk-list govuk-error-summary__list">
+            <li th:text="#{'login.' + ${error}}">Error Message</li>
+          </ul>
+        </div>
+      </div>
+
+      <div th:if="${param.success}" class="govuk-warning-text">
+        <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
+        <strong class="govuk-warning-text__text">
+          <span class="govuk-warning-text__assistive">Success</span>
+          <span th:text="#{'login.' + ${param.success}}"></span>
+        </strong>
+      </div>
+
+      <form th:action="@{/login.do}"
+            method="post"
+            novalidate="novalidate"
+            accept-charset="UTF-8">
+
+        <div class="govuk-form-group"
+             th:each="prompt,iter : ${prompts}">
+          <label class="govuk-label"
+                 th:attr="for=${prompt.key}"
+                 th:text="${prompt.value[1] == 'Email' ? 'Email address' : prompt.value[1]}">
+          </label>
+          <input th:name="${prompt.key}"
+                 th:type="${prompt.value[0]}"
+                 th:attr="autocomplete=${(prompt.value[0] == 'password') ? 'off' : null}"
+                 class="govuk-input"/>
+        </div>
+
+        <input th:if="${session.SPRING_SECURITY_SAVED_REQUEST}"
+               th:unless="${form_redirect_uri}"
+               th:name="form_redirect_uri"
+               th:type="hidden"
+               th:value="${session.SPRING_SECURITY_SAVED_REQUEST.redirectUrl}"
+               type="hidden"/>
+        <input th:if="${form_redirect_uri}"
+               th:name="form_redirect_uri"
+               th:type="hidden"
+               th:value="${form_redirect_uri}"
+               type="hidden"/>
+        <input th:if="${param.login_hint}"
+               th:unless="${login_hint}"
+               th:name="login_hint"
+               th:type="hidden"
+               th:value="${param.login_hint}"
+               type="hidden"/>
+        <input th:if="${login_hint}"
+               th:name="login_hint"
+               th:type="hidden"
+               th:value="${login_hint}"
+               type="hidden"/>
+
+        <button class="govuk-button">Continue</button>
+      </form>
+
+      <p class="govuk-body" th:unless="${#strings.isEmpty(links['forgotPasswordLink'])}">
+        <a href="/forgot_password"
+           th:href="@{${links['forgotPasswordLink']}}"
+           class="govuk-link">Forgot your password?</a>
+      </p>
+    </div>
+
+    <div class="govuk-grid-column-full">
+      <hr class="govuk-section-break govuk-section-break--m">
+    </div>
+
+    <% if p("sso_enabled") == "enabled" %>
+      <div class="govuk-grid-column-full">
```

Comment:
Instead of a bosh job property we can use thymeleaf here

```suggestion
      <div class="govuk-grid-column-full" th:unless="${oauthLinks.isEmpty()}">
```

`oauthLinks` is `Map<String, String>`so we can check the length.

This means we don't have to keep the bosh properties in sync with how UAA is configured